### DISCUSSION TITLE
Does the Encode Plus need to be added via Schlage app instead of HomeKit?

### DISCUSSION THREAD

**Discussion by OwenVey** [dknowles2/ha-schlage#76] [Q&A] [answered]
I have two Encode Plus locks and I initially set them up via HomeKit. When adding this integration, my locks are not showing up so I'm wondering if they need to be added the Schlage app initially?

**Comment by OwenVey** [comment, marked as answer]
Confirmed this myself by deleting the lock out of HomeKit and adding it via the Schlage app. Then I just had to reload this Integration and it worked!

**Comment by dknowles2** [comment]
Yes I believe that if you add it directly to HomeKit it will side-step the Schlage API. Since this integration only communicates directly with the Schlage API it wouldn't have any knowledge of the lock in that case.
